### PR TITLE
feat: [110] add error pages

### PR DESCRIPTION
**Pull request by Cihatata**
Issues #110
When I looked at the project, I saw this as an entry-level issue and wanted to do it.

I think page not found(404) and no authorization(403) pages should be separate. They mean different things and the url names should be different. That's why I made it as two different pages. I separated the texts in the design.
image

### CODE REVIEW

**Review by uparkalau — CHANGES_REQUESTED** [review]
It looks good . Could you please add a test using [vitest ](https://vitest.dev/)
It is under test folder frontend\src\tests

File: frontend/src/scenes/errors/constant.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const TEXT_404 = 'The URL doesn’t exist';
```
Comment (erenfn) [inline frontend/src/scenes/errors/constant.js:1]: constants should be under src/utils folder. scenes folder is just for the pages

**Review by erenfn — CHANGES_REQUESTED** [review]

**Comment by Cihatata** [comment]
> It looks good . Could you please add a test using [vitest ](https://vitest.dev/) It is under test folder frontend\src\tests

I added test cases

**Review by erenfn — DISMISSED** [review]

**Comment by Cihatata** [comment]
Is there anything expected from me for this PR?

**Comment by uparkalau** [comment]
> Is there anything expected from me for this PR?

Yes please delete jest library and use vitest as I requested in previous comment.

File: frontend/vite.config.js
Comment (uparkalau) [inline frontend/vite.config.js:1]: We do not need extra library for this kind of tests, delete setup files , use vitest. https://vitest.dev/guide/

**Review by uparkalau — CHANGES_REQUESTED** [review]
Please, mark all conversation resolved when you done

**Comment by uparkalau** [comment]
> Is there anything expected from me for this PR?

How is it going?

**Comment by Cihatata** [comment]
> > Is there anything expected from me for this PR?
> 
> How is it going?

I got errors with toHaveStyle when I was doing the job, so I added the setupTest file. I'll see if there's a different way.

**Comment by Cihatata** [comment]
Fixed your comments @uparkalau

**Review by erenfn — DISMISSED** [review]
Do git pull from develop and it is good to go

**Comment by uparkalau** [comment]
IT looks good. Accept new changes from the develop branch and you can merge it.

**Review by uparkalau — APPROVED** [review]

**Review by uparkalau — APPROVED** [review]

**Comment by Cihatata** [comment]
> Do git pull from develop and it is good to go
updated branch

**Comment by uparkalau** [comment]
> > It looks good . Could you please add a test using [vitest ](https://vitest.dev/) It is under test folder frontend\src\tests
> 
> I added test cases

Yes, you can merge it.
Thank you.

**Comment by Cihatata** [comment]
> > > It looks good . Could you please add a test using [vitest ](https://vitest.dev/) It is under test folder frontend\src\tests
> > 
> > 
> > I added test cases
> 
> Yes, you can merge it. Thank you.

Thank you, I am waiting approve from @erenfn

**Review by erenfn — APPROVED** [review]

**Comment by Cihatata** [comment]
I cant show Merge button I guess I don't have access. Can you merge it?
image

**Pull request merged**